spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -170,18 +171,29 @@ def __init__(self, parent=None, background_color=None):
         self.thumbnails_sb.sig_redirect_stdio_requested.connect(
             self.sig_redirect_stdio_requested)
 
+        # Widget empty pane
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "plots",
+            _("You haven't generated any plots yet."),
+            _("Run plot generating code in the Editor or IPython console to "
+              "see your plots appear here. This pane only supports Matplotlib "
+              "plots.")
```

Comment:
It seems likely that users could be confused here, since many, many packages generate Matplotlib-compatible plots (Pandas, Seaborn, etc., etc.) whereas users might assume that their plots won't work unless they create them directly with Matplotlib itself. I'd suggest either revising the message like this, to be less restrictive as well as more concise:

```suggestion
              "see your Matplotlib-compatible figures appear here.")
```

Or simply eliminate that part:

```suggestion
              "see your figures appear here.")
```

Or, perhaps be more specific as to what's actually widely used that's not supported, e.g. Jupyter-specific visualizations that require a web browser environment?

Also, substitute "figures" for "plots to avoid repetition and be slightly more precise.

Reply:
> Or, perhaps be more specific as to what's actually widely used that's not supported, e.g. Jupyter-specific visualizations that require a web browser environment?

Sure, that would be better (i.e. no Bokeh, Plotly, Altair plots). And you're right about mentioning Matplotlib. What we can actually display on this pane are static images (i.e. png, jpeg or svg ones) generated by code.

However, I don't know how to describe that in simple terms. So, we'd appreciate your help with it.

Reply:
Okay, understood. I wasn't quite sure how to best describe it either, but given that, the best I could come up with was something like:

```suggestion
              "see your figures appear here. This pane currently supports "
              "static images, though not interactive Jupyter-specific plots "
              "like Bokeh, Plot.ly or Altair.")
```

Reply:
All SGTM, thanks :+1: